javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,89 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+const productList = document.querySelector("ul");
+
+function renderProducts(products) {
+    // your code here
+    productList.innerHTML = "";
+    products.forEach(product => {
+        let li = document.createElement("li");
+        let title = document.createElement("span");
+        let productPrice = document.createElement("div");
+        let productRating = document.createElement("div");
```

Comment:
Just a suggestion to think about and not needed for this homework, but can you think of a way to do this without recreating the elements every time. Creating elements takes a lot of processing. Could you use CSS to show and hide the elements instead of deleting and creating them.

Reply:
Thanks for the suggestion. I've read about it. It was very interesting.